src/App.jsx

```diff
@@ -71,39 +72,34 @@ const initialState = {
 };
 
 function App() {
-  const initialBlogs = initialTitles.map(title => titleToBlog(title));
   const [state, runAction] = useReducer(stateManagementFunction, initialState);
 
   useEffect(() => {
-    setTimeout(() => {
-      const blogTitlesFromLocalStorage =
-        getItem(LOCALSTORAGE_KEY) || initialBlogs;
+    runAction({
+      type: 'START_LOADING_BLOG_TITLES',
+    });
+    listRecords({ searchText: state.searchText }).then(loadedTitles => {
```

Comment:
Getting the initial blog records is like running an empty search! So instead of having two separate useEffects, one for getting the blog titles on load and one for running the search I combined the behavior so that this effect will run when the app renders with `searchText=''` and whenever the search text changes and a real search needs to be run.

Reply:
Sorry I have a dumb question, but I found it is so hard to read these Github changes , colors and shades and spaces ... When you do not commit it to main and leave it as a pull request like this, is this the only way I can look at it?

Reply:
@Miminiverse not dumb at all, totally valid point!

I find the PRs so much easier to read in "Split View". When you're in the "files changed" tab click on the settings icon in the top bar and choose split view
![image](https://user-images.githubusercontent.com/6400323/225894462-0325bd71-051e-41d4-8a5a-c6e22093f231.png)

Let me know if this helps, we can explore other ways to review code changes if not.

Reply:
Good morning, yep I have tried it and it helps a little bit, I have searched around and looks like there's some extensions that may help, or maybe I'm just not getting used to reading these, and it'll be better after I read more  😀